app/assets/javascripts/misc/recovery-page-controller.js

```diff
@@ -42,7 +42,7 @@ function handleSubmit(event) {
   const invalidMatches = inputs.reduce(function(accumulator, input, index) {
     const value = input.value;
 
-    if (value === recoveryWords[index].innerHTML.replace(/\s+/, '')) {
+    if (value.toUpperCase() === recoveryWords[index].innerHTML.replace(/\s+/, '').toUpperCase()) {
```

Comment:
I figured there might still be transcription errors, but fair enough about re-implementing. Case-insensitivity is probably enough for a first pass and we can wait for user feedback/research maybe?